src/populate_result_buffer.jl

```diff
@@ -142,9 +121,10 @@ function ChunkedBase.populate_result_buffer!(
             if Parsers.eof(code) && !(col_idx == N && Parsers.delimited(code))
                 row_status |= RowStatus.TooFewColumns
                 row_status |= RowStatus.HasColumnIndicators
+                added_collind_row || (added_collind_row = true; addrows!(colinds))
                 for _col_idx in col_idx:N
                     skip_element!(cols[_col_idx])
-                    mark_missing!(colinds, colinds_row_idx, _col_idx)
+                    colinds[colinds_row_idx, _col_idx] = true
```

Comment:
Just to confirm, I assume `@inbounds` here wouldn't add much because of the `for` loop.

Reply:
This entire block should be using `@inbouds` from the outer loop: `@inbounds for type_enum in enum_schema`